js/models/gitRepo.js

```diff
@@ -0,0 +1,26 @@
+(function(module) {
+  var gitRepos = {};
+
+  gitRepos.all = [];
+
+  gitRepos.requestRepos = function(callback) {
+    $.ajax({
+      url: 'https://api.github.com/users/' + vienGitToken.user + '/repos' + '?per_page=10' + '&sort=updated' + '&since=2016-05-09T00:00:01Z',
+      type: 'GET',
+      headers: {'Authorization':'token ' + vienGitToken.token},
+      success: function(data, message, xhr) {
+        console.log(data);
+        gitRepos.all = data;
+        callback();
+      }
+    });
+  };
+
+  gitRepos.with = function(attr) {
```

Comment:
Don't just copy-paste code from the pair assignment. Try to think of what your portfolio needs and modify it to suit.